src/frontend/components/TranscriptReader/index.tsx

```diff
@@ -31,8 +31,22 @@ export const TranscriptReader = ({ transcript }: TranscriptReaderProps) => {
     parser.flush();
   }, []);
 
+  const transcriptWrapperRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (!transcriptWrapperRef.current) return;
+
+    transcriptWrapperRef.current
+      .querySelector(`.active-sentence`)
+      ?.scrollIntoView({
+        behavior: 'smooth',
+        block: 'nearest',
```

Comment:
Can you change it to `center` please ?

Reply:
fixed e5a95830f63393e8bcf0ab8297269290bcce6ba6